examples/self_managed_node_group/main.tf

```diff
@@ -97,6 +97,12 @@ module "eks" {
 
   self_managed_node_group_defaults = {
     create_security_group = false
+
+    # enable discovery of autoscaling groups by cluster-autoscaler
+    autoscaling_group_tags = {
+      "k8s.io/cluster-autoscaler/enabled" : true,
+      "k8s.io/cluster-autoscaler/${local.name}" : true,
```

Comment:
per https://docs.aws.amazon.com/eks/latest/userguide/autoscaling.html should this be `owned` ?